api/Window.json

```diff
@@ -4625,12 +4625,42 @@
             "edge": {
               "version_added": false
             },
-            "firefox": {
-              "version_added": "15"
-            },
-            "firefox_android": {
-              "version_added": "15"
-            },
+            "firefox": [
```

Comment:
FYI, this matches the windows event handler versioning to the event itself - ie puts it behind the preference. To me this seems like it was an omissions, since the handler isn't usable without the event.